src/commands/check.ts

```diff
@@ -12,34 +19,30 @@ export const data = new SlashCommandBuilder()
   );
 
 export async function execute(interaction: CommandInteraction) {
-  if (!interaction.isChatInputCommand()) {
-    return;
-  }
+  if (!interaction.isChatInputCommand()) return;
 
   await interaction.deferReply({ ephemeral: true });
 
-  logger.info(`running check command (user.id=${interaction.user.id})`);
```

Comment:
why remove these logging statements?

Reply:
I was just trying to clean up some of the logs we use throughout the discord bot. I don't think they serve too much purpose atm in terms of debugging. I should remove the others for report.ts